tests/conftest.py

```diff
@@ -144,7 +144,7 @@ def chdir_tmp_path(tmp_path):
 # CMC 2024-03-01 this doesn't need to be a fixture
 @pytest.fixture
 def keccak():
-    return Web3.keccak
+    return keccak256
```

Comment:
we should probably also have a sanity check unit test, like for the constant `keccak256(b"")` case should return the well known result `c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470` (https://ethereum.stackexchange.com/a/107985)

Reply:
I've added a todo to remove the fixture in https://github.com/vyperlang/vyper/pull/3846
I would prefer to remove the fixture after that one lands to avoid many conflicts